.github/workflows/generate-rst.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate cells docs
+
+on:
+    push:
+    pull_request:
+
+jobs:
+
+  generate-cells:
+    runs-on: ubuntu-18.04
+    steps:
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Generate cells READMEs
+        run: |
+          sudo apt update
```

Comment:
These should all be done in the conda environment.

Reply:
switched to conda environment